### DISCUSSION TITLE
Persistent Storage for Windows

### DISCUSSION THREAD

**Discussion by trymeouteh** [ventoy/Ventoy#1236] [Ideas]
Please allow one to create persistent storage installations of Windows onto their USB and portable harddrives. This will be useful for non-windows users to do stuff on Windows without having to dual boot windows or get another computer for Windows.

**Comment by ventoy** [comment]
Do you mean Windows VHD(x) ?
https://www.ventoy.net/en/plugin_vhdboot.html

**Reply by trymeouteh** [reply]
By having Windows installed on a USB stick or harddrive and be able to boot up Windows, and do work on it like install apps, create files, etc and when I boot the USB stick or drive down, the changes made will still be remained the next time I boot it up. Essentially the ability to make a portable OS on a stick.

**Reply by ventoy** [reply]
VHD(x)  WinToGo can do so.

**Reply by trymeouteh** [reply]
Is there a way on Linux with open source software to create a VHD file?

Edit: VirtualBox can create VHD files on Linux
